docker/kubernetes/helm/templates/api/deployment.yaml

```diff
@@ -4,7 +4,7 @@ metadata:
   name: {{ printf "%s-%s" .Release.Name "api"  }}
   namespace: {{ include "common.names.namespace" . | quote }}
   labels: {{- include "common.labels.standard" . | nindent 4 }}
-    app.kubernetes.io/component: novu-api
+    app.kubernetes.io/component: {{ printf "%s-%s" .Release.Name "api"  }}
```

Comment:
These changes belongs to other PR.

Reply:
oh crap that was not intentional, apparently I made a mistake. Should I open a new PR or can you pick it up only in part?

Reply:
Fixed. 😄